marklogic-data-hub-central/ui-custom/src/views/Detail.tsx

```diff
@@ -35,6 +35,12 @@ const COMPONENTS = {
   ImageGalleryMulti: ImageGalleryMulti,
   Membership: Membership
 };
+const EXPANDIDS = {
```

Comment:
When I leave the Detail page and then return to the Detail page, any collapsed sections become expanded again. Can we persist the state of the sections across visits to the Detail page? 

Also, for naming consistency, let's name the EXPANDID properties the same names as the detail properties in config.json. Let's also order them the same way:
```
  membership: true,
  info: true,
  relationships: true,
  imageGallery: true
```